src/python/pants/backend/python/rules/pex.py

```diff
@@ -409,22 +420,27 @@ async def two_step_create_pex(two_step_pex_request: TwoStepPexRequest) -> TwoSte
     req_pex_name = "__requirements.pex"
 
     # Create a pex containing just the requirements.
-    requirements_pex_request = PexRequest(
-        output_filename=req_pex_name,
-        requirements=request.requirements,
-        interpreter_constraints=request.interpreter_constraints,
-        # TODO: Do we need to pass all the additional args to the requirements pex creation?
-        #  Some of them may affect resolution behavior, but others may be irrelevant.
-        #  For now we err on the side of caution.
-        additional_args=request.additional_args,
-    )
-    requirements_pex = await Get[Pex](PexRequest, requirements_pex_request)
+    if request.requirements.requirements:
+        requirements_pex_request = PexRequest(
+            output_filename=req_pex_name,
+            requirements=request.requirements,
+            interpreter_constraints=request.interpreter_constraints,
+            # TODO: Do we need to pass all the additional args to the requirements pex creation?
+            #  Some of them may affect resolution behavior, but others may be irrelevant.
+            #  For now we err on the side of caution.
+            additional_args=request.additional_args,
+            description=f"Resolving {', '.join(request.requirements.requirements)}",
```

Comment:
This is repetitive of the default that would be computed, but is repeated to emphasize that we purposely want this value (even if the default changes) since we know with certainty that resolving is what this pex call is doing.